pom.xml

```diff
@@ -30,7 +30,7 @@
 
     <properties>
 			<openmrs.platform.version>2.1.6</openmrs.platform.version>
-			<openmrs.fhir2.version>1.0.0-SNAPSHOT</openmrs.fhir2.version>
+			<openmrs.fhir2.version>1.3.0</openmrs.fhir2.version>
```

Comment:
would there be any issues with 1.4?

Reply:
Yes we should move to fhir2 1.4. We can play this as a part of other backend story.